pkgs/development/python-modules/labgrid/default.nix

```diff
@@ -48,6 +49,7 @@ buildPythonPackage rec {
     packaging
     pexpect
     pyserial
+    pytest
```

Comment:
Why does this need pytest?

Reply:
We use the pytest directive to enforce a minimum pytest version within labgrid. But this can be removed for NixOS since we now pytest is new enough.

Reply:
Pytest is now replaced with an empty line.